Python Basics/python_basics_home_task.py

```diff
@@ -0,0 +1,38 @@
+# Home task for Module 1 - Python Basics
+# Create list of 100 random numbers from 0 to 1000
+import random  # Import of random module for using of random function
+
+list_of_random = []  # Define the empty list
+for j in range(1000):  # Define of FOR statement with range of iterations from 0 to 999
+    list_of_random.append(random.randint(0, 999))  # Adding of randomly generated by randint function INT values to list
+print('List with generated random values:', list_of_random)  # Printing final list to console
+# Sort list from min to max (without using sort())
+for run in range(len(list_of_random) - 1):  # Define of initial FOR statement with range of iterations minus 1 from list
+    for i in range(len(list_of_random) - 1 - run):  # Inner FOR statement with decreasing iterations for each run
+        if list_of_random[i] > list_of_random[i + 1]:  # Compare two adjacent values from list
+            list_of_random[i], list_of_random[i + 1] = list_of_random[i + 1], list_of_random[i]  # Swap places
+print('Sorted from min to max list:', list_of_random)  # Printing of sorted by bubble sorting list
+# Calculate average for even and odd numbers
+even_numbers = []  # Define empty list for even numbers
+odd_numbers = []  # Define empty list for odd numbers
+for i in range(len(list_of_random)):  # Define FOR statement with number of iterations equal to list length
+    if list_of_random[i] % 2 == 0:  # Check if number can be divided by 2 without reminder
+        even_numbers.append(list_of_random[i])  # Adding of even values to separate list
+    else:
+        odd_numbers.append(list_of_random[i])  # Else other values added to odd values list
```

Comment:
2 things here:
1) If you want to iterate through iterable collection, you do not need to use its indeses each time. Simple 'for' loop can also take element itself. For example:
```suggestion
for number in list_of_random:
    if number % 2 == 0:  # Check if number can be divided by 2 without reminder
        even_numbers.append(number)  # Adding of even values to separate list
    else:
        odd_numbers.append(number)  # Else other values added to odd values list
```
2) There are another options how to do it, it's not required to change current implementation, but just FYI: you can use list comprehension or 'filter' function as well. If you're interested you can find such examples in the Internet.